Libraries/libbom/Tools/lsbom.cpp

```diff
@@ -294,6 +294,68 @@ extern "C" {
     } __attribute__((packed));
 }
 
+/**
+ *  Todo:
+ *      Group Name
+ *      PermissionsText
+ *      FileSizeFormatted
+ *      ModificationTimeFormatted
+ *      Username
+ *      UserGroupName
+*/
+std::string
+printItemToString(Options::PrintItem item, std::string path, struct bom_path_info_2 *info){
+
+    switch (item) {
+        case Options::PrintItem::Checksum:
+            if (info->type == bom_path_type_directory) {
+                return "";
+            }
+            return std::to_string(ntohl(info->checksum));
+        case Options::PrintItem::FileName:
+            return path;
+        case Options::PrintItem::FileNameQuoted:
+            return "\"" + path + "\"";
+        case Options::PrintItem::GroupID:
+            return std::to_string(ntohl(info->group));
+        case Options::PrintItem::GroupName:
+
+        break;
```

Comment:
For these unfinished ones, I'd do it like this:

    case Options::PrintItem::SomeCase:
        return std::string(); // TODO: implement